llvm/test/CodeGen/NVPTX/tcgen05-mma-block-scale-ptx88.ll

```diff
@@ -0,0 +1,526 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -o - -mcpu=sm_100a -march=nvptx64 -mattr=+ptx88 | FileCheck %s
+; RUN: llc < %s -o - -mcpu=sm_101a -march=nvptx64 -mattr=+ptx88 | FileCheck %s
+; RUN: %if ptxas-12.9 %{ llc < %s -march=nvptx64 -mattr=+ptx88 -mcpu=sm_100a | %ptxas-verify -arch=sm_100a %}
+; RUN: %if ptxas-12.9 %{ llc < %s -march=nvptx64 -mattr=+ptx88 -mcpu=sm_101a | %ptxas-verify -arch=sm_101a %}
```

Comment:
I think ptxas-related lit features had changed recently and you may need to update these RUN lines.

Also, given that sm_101 has been renamed to sm_110 by NVIDIA, the last line will fail if the test is run with ptxas from cuda-13, so that needs to be constrained, as well, and probably needs another line for testing with cuda-13.